client/gutenberg/editor/controller.js

```diff
@@ -61,28 +53,6 @@ function getPostID( context ) {
 	return parseInt( context.params.post, 10 );
 }
 
-function waitForSiteIdAndSelectedEditor( context ) {
```

Comment:
Removing this function because it waits for editor preference (`selectedEditor`) to determine which editor to redirect to.